depends/hosts/darwin.mk

```diff
@@ -0,0 +1,17 @@
+OSX_MIN_VERSION=10.7
+OSX_SDK_VERSION=10.9
```

Comment:
(This is more of a general reference note.)

I'm not sure offhand how this is determined, other than referencing Wikipedia. (I assume there's something in the dev docs but I can't access them right now.) I think Xcode 6.3 will bump this to 10.10.